---
title: "GIMiCC"
author: "Steven C. Pike"
date: "`r Sys.Date()`"
output: rmarkdown::html_vignette
vignette: >
    %\VignetteIndexEntry{GIMiCC}
    %\VignetteEngine{knitr::rmarkdown}
    %\VignetteEncoding{UTF-8}
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(cache = FALSE, warning = FALSE, message = FALSE, 
                        cache.lazy = FALSE,collapse = TRUE, comment = "#>"
)
```

**Loading package:**  
```{r eval=TRUE}
library(GIMiCC)

```
The GIMiCC package contains reference libraries derived from 
Illumina HumanMethylation450K DNA methylation 
microarrays.

The reference libraries were used to estimate proportions of 18 cell types 
(tumor, endothelial, stromal, astrocyte, oligodendrocyte, microglia, 
GABAergic neurons,glutamatergic neurons, neutrophil, monocyte, B naïve, 
B memory, CD4T naïve, CD4T memory, CD8T naïve, CD8T memory, T regulatory, 
and natural killer cells) for 4 molecular subtypes of glioma using a modified 
version of the algorithm constrained projection/quadratic programming described 
in Houseman et al. 2012. 

**Objects included:**  

*GIMiCC_deconvolution function for tumor microenvironment deconvolution:* 
 
 We offer the function GIMiCC_deconvolution to estimate proportions for 18 
 cell types (tumor, endothelial, stromal, astrocyte, oligodendrocyte, microglia, 
 GABAergic neurons,glutamatergic neurons, neutrophil, monocyte, B naïve, 
 B memory, CD4T naïve, CD4T memory, CD8T naïve, CD8T memory, T regulatory, 
 and natural killer cells) for 4 molecular subtypes of glioma using a modified 
 version of the algorithm constrained projection/quadratic programming described 
 in Houseman et al. 2012. 
 *see ?HiTIMED_deconvolution for details*  
```{r eval=TRUE}
# Step 1: Load example data
library(ExperimentHub)
Example_Beta<-query(ExperimentHub(), "GIMiCC")[["EH????????????"]]


# Step 2: Run HiTIMED and show results
GIMiCC_results <- GIMiCC_Deconvo(Example_Beta, tumor.type = "GBM", h = 5)
head(GIMiCC_results)

```

```{r}
sessionInfo()
```


**References**

Z Zhang, LA Salas et al. (2023) HiTIMED: hierarchical tumor immune 
microenvironment epigenetic deconvolution for accurate cell type resolution 
in the tumor microenvironment using tumor-type-specific DNA methylation data.
Journal of Tranlsational Medicine, 8;20(1):516. 
doi: [10.1186/s12967-022-03736-6] 
(https://dx.doi.org/10.1186/s12967-022-03736-6).


Zheng X et al. (2017). Estimating and accounting for tumor 
purity in the analysis of DNA methylation data from cancer studies. Genome Biol. 
2017;18(1):17. doi: [10.1186/s13059-016-1143-5]
(https://doi.org/10.1186/s13059-016-1143-5). 


LA Salas et al. (2018). An optimized library for reference-based deconvolution
of whole-blood biospecimens assayed using the Illumina HumanMethylationEPIC 
BeadArray. Genome Biology 19, 64. doi: [10.1186/s13059-018-1448-7]
(https://dx.doi.org/10.1186/s13059-018-1448-7).


DC Koestler et al. (2016). Improving cell mixture deconvolution
by identifying optimal DNA methylation libraries (IDOL)}. BMC bioinformatics.
doi: [10.1186/s12859-016-0943-7]
(https://dx.doi.org/10.1186/s12859-016-0943-7)


LA Salas et al. (2022). Enhanced cell deconvolution of peripheral blood using 
DNA methylation for high-resolution immune profiling. 
Nature Communications 13(1):761. 
doi:[10.1038/s41467-021-27864-7](https://dx.doi.org/10.1038/s41467-021-27864-7).  


EA Houseman et al. (2012) DNA methylation arrays as surrogate
measures of cell mixture distribution. BMC Bioinformatics 13, 86.
doi: [10.1186/1471-2105-13-86](https://dx.doi.org/10.1186/1471-2105-13-86).  

